### PR TITLE
Liquibase neo4j 117

### PR DESCRIPTION
**Pull request by clarenced**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,7 +64,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.enforcer.requireJavaVersion>${maven.compiler.source}</maven.enforcer.requireJavaVersion>
-        <liquibase.version>4.7.1</liquibase.version>
+        <liquibase.version>0-SNAPSHOT</liquibase.version>
```
Comment (fbiville) [inline pom.xml:67]: Note: need to get https://github.com/liquibase/liquibase/pull/2569/ merged and released first

File: src/main/java/liquibase/ext/neo4j/parser/FormattedCypherChangeLogParser.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package liquibase.ext.neo4j.parser;
+
+import liquibase.parser.core.formattedsql.FormattedSqlChangeLogParser;
+
+public class FormattedCypherChangeLogParser extends FormattedSqlChangeLogParser {
+
+  @Override
+  protected boolean supportExtension(String changelogFile){
```
Comment (fbiville) [inline src/main/java/liquibase/ext/neo4j/parser/FormattedCypherChangeLogParser.java:8]:
```suggestion
  protected boolean supportsExtension(String changelogFile){
```

**Comment by fbiville** [comment]
@clarenced il manque le fichier `0003-update-rating.cypher` :)

**Comment by clarenced** [comment]
@fbiville Ok je vais le rajouter dans le commit

**Comment by fbiville** [comment]
> @fbiville Il était dans présent dans ce commit [b7606fd#diff-3f021a26ed6bbadde7dbdf9cf3d336a25b59b0c4e540d93ed3b7afbfd42ea130](https://github.com/liquibase/liquibase-neo4j/commit/b7606fd7feb6b1ec13bd624f846702e7012f8b57#diff-3f021a26ed6bbadde7dbdf9cf3d336a25b59b0c4e540d93ed3b7afbfd42ea130)

Yep, all good. I should not comment that late in the evening :)

**Comment by fbiville** [comment]
@clarenced still waiting for a release first

**Comment by fbiville** [comment]
No push rights on this PR for me, so the rebased version lives here: https://github.com/liquibase/liquibase-neo4j/pull/161

**Pull request closed without merging**